wurst/util/Colors.wurst

```diff
@@ -219,6 +296,15 @@ init
 	assertTrue(fromHexString("33ccff44") == colorA(204, 255, 68, 51))
 
 	assertTrue(PLAYER_COLOR_RED != null)
+	
+@Test public function testHSV()
+	colorHSV(0, 0, 1).test()
```

Comment:
Can you add an actual use case? Right now I don't see what you would use it for.

Reply:
Use what for? colorHSV in general or the tests/asserts?

Reply:
Something that someone would actually use in a map. E.g. have a rgb color, convert it and decrease saturation, then convert back. The test seems very artificial right now.

Reply:
Instead of typing tons of manual conversions (as HSVtoRGB and vice versa has alot of edge cases) this checks both and is alot shorter. Proving that both are working as intended. I don't see the issue as this is the most efficient way while using 50 lines less

Reply:
Yea, but I'm not talking about asserting correctness.